js/react_native/package.json

```diff
@@ -60,29 +62,13 @@
   ],
   "description": "ONNX Runtime bridge for react native",
   "repository": "https://github.com/Microsoft/onnxruntime.git",
-  "react-native-builder-bob": {
-    "source": "lib",
-    "targets": [
-      "commonjs",
-      "module",
-      [
-        "typescript",
-        {
-          "project": "tsconfig.build.json",
-          "tsc": "../node_modules/.bin/tsc"
-        }
-      ]
-    ],
-    "output": "dist"
-  },
   "dependencies": {
     "@expo/config-plugins": "^7.2.4",
     "buffer": "^6.0.3",
     "onnxruntime-common": "file:../common"
   },
   "scripts": {
     "typescript": "tsc --noEmit",
-    "prepare": "bob build",
```

Comment:
Could you please describe this change in the PR's descriptions?

Reply:
this `bob` was part of "react-native-builder-bob", see line 63 above. It is no longer being used.